test/models/post_test.rb

```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class PostTest < ActiveSupport::TestCase
+  def setup
+    @post = Post.new(content: "I'm a post :)")
+  end
+
+  # validate post content
+  test "post content should be present (nonblank)" do 
+    @post.content = "    " * 10
+    assert_not @post.valid?
+  end
+
+  test "post content should have a minimum length" do 
+    @post.content = "a" * 4
+    assert_not @post.valid?
```

Comment:
I cannot see any length validations for post.

Can you double check what is the reason of this object being not valid? Cause definitely it is not its content.

Reply:
We changed this as well, had not completed these tests